protocol/events.proto

```diff
@@ -41,6 +47,7 @@ message Event {
     Sleep sleep = 12;
     Send send = 13;
     Stop stop = 14;
+    Compensation compensation = 15;
```

Comment:
should not that be optional? I would rather consider compensation at simple Event with an additional metadata instead = if we are at X events, then I expect X compensations and there is no need to link if the numbers match

Reply:
ok!!